glass/fields.py

```diff
@@ -140,16 +130,21 @@ def multalm(alm: Alms, bl: npt.NDArray[typing.Any], *, inplace: bool = False) ->
     return out
 
 
-def transform_cls(cls: Cls, tfm: ClTransform, pars: tuple[typing.Any, ...] = ()) -> Cls:
+def transform_cls(
+    cls: npt.NDArray[np.float64],
+    tfm: str | typing.Callable[[npt.NDArray[np.float64]], npt.NDArray[np.float64]],
+    pars: tuple[typing.Any, ...] = (),
+) -> list[list[float] | npt.NDArray[np.float64]]:
     """Transform Cls to Gaussian Cls."""
     gls = []
     for cl in cls:
-        if cl is not None and len(cl) > 0:  # type: ignore[redundant-expr]
+        if len(cl) > 0:
```

Comment:
`None` should be accepted here for backward compatibility

Reply:
As discussed, I'm deprecating the `None` option. But have restored the type as `int | tuple[int, ...]`